subsys/bluetooth/audio/ascs.c

```diff
@@ -2451,7 +2456,7 @@ static ssize_t ascs_cp_write(struct bt_conn *conn,
 	}
 
 respond:
-	bt_gatt_notify(ascs->conn, ascs->control_point_attr, rsp_buf.data, rsp_buf.len);
+	bt_gatt_notify(ascs->conn, control_point_attr(), rsp_buf.data, rsp_buf.len);
```

Comment:
I would suggest to simply use `bt_gatt_notify_uuid` with `BT_UUID_ASCS_ASE_CP` instead, which seems to do just about the same thing, but at the GATT layer instead. That way we can omit the `control_point_attr` and `control_point_attr_cb` functions

Reply:
Hmm, I see the advantage of using `bt_gatt_notify_uuid` in terms of removing of code, while in terms of optimization  `bt_gatt_notify_uuid` iterates the db every time it's used

Reply:
I've changed to use bt_gatt_notify_uuid, and moved the CP attribute to be the second entry in attrs array.
Anyway, the `&ascs_svc.attrs[1]` could be used here IMO instead